internal/session/connection_test.go

```diff
@@ -42,13 +43,15 @@ func TestConnection_Create(t *testing.T) {
 				clientTcpPort:      22,
 				endpointTcpAddress: "127.0.0.1",
 				endpointTcpPort:    2222,
+				userClientIp:       "127.0.0.1",
```

Comment:
nit: Might be good to test a different address to ensure what is coming back comes from this field.

Reply:
ty!